plugins/aws/neptune/neptuneDBIamAuth.js

```diff
@@ -0,0 +1,55 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'Neptune Database IAM Authentication Enabled',
+    category: 'Neptune',
+    domain: 'Databases',
+    severity: 'Medium',
+    description: 'Ensure that AWS Neptune database instance has IAM database authentication feature enabled.',
+    more_info: 'Enabling IAM authentication for AWS Neptune adds an extra layer of security by allowing access control through IAM credentials, providing more precise control over who can access your Neptune resources.',
```

Comment:
```suggestion
    more_info: 'Enabling IAM authentication for AWS Neptune adds an extra layer of security by allowing access control through IAM credentials. It ensures that network traffic for clusters is encrypted using SSL and allows centralized management. All authentication requests are automatically signed with a secure access key instead of using a password.',
```